ERCS/erc-xx.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Operator contract for non delegated EOAs
+description: A standard for operating batch executions on behalf of non delegated EOAs
+author: Marcelo Morgado (@marcelomorgado), Manoj Patidar (@patidarmanoj10)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians linking to this PR.
https://ethereum-magicians.org/c/ercs/57